### PR TITLE
TW-1090 RFA viewer elevate workspace role

### PR DESCRIPTION
**Pull request by claireriley**
Please note on "new" page changes have been made to:
* intro (edited)
* #contents (edited)
* #requesting-role-access-for-workspaces (totally new)
* #approving-requests-for-access (edited)

### CODE REVIEW

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/collaborating-in-postman/requesting-access-to-elements.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  - type: subtitle
+    name: "Blog posts"
+  - type: link
+    name: "2 Ways to Access and Edit a Postman Element"
+    url: "https://blog.postman.com/access-and-edit-postman-element/"
+warning: false
+
+---
+
+By default, you can view [workspaces](/docs/collaborating-in-postman/using-workspaces/creating-workspaces/) and the Postman elements in them that are shared with your team. If you need to edit an element, such as a workspace, collection, API, environment, mock server, or monitor, you can request [Editor role](/docs/collaborating-in-postman/roles-and-permissions/) access. If you need to manage a workspace, you can also request [Admin role](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles) access.
```
Comment (ramji367) [inline src/pages/docs/collaborating-in-postman/requesting-access-to-elements.md:31]: @claireriley - You can't explicitly request Admin role in a workspace. They need to submit a request to edit. When approving, Workspace Admin can choose to grant the Editor role or the Admin role

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/pages/docs/collaborating-in-postman/requesting-access-to-elements.md (hunk 1 above)
Comment (claireriley) [reply src/pages/docs/collaborating-in-postman/requesting-access-to-elements.md]: Okay was going off of the ability to note it in the description, but will remove

**Review by ramji367 — COMMENTED** [review]
Looks good!

**Pull request merged**